lin_actuator.h

```diff
@@ -0,0 +1,36 @@
+/* 
+ * File:   LINAC.h
+ * Author: zx100
+ *
+ * Created on February 10, 2019, 12:25 AM
+ */
+
+#ifndef LINAC_H
+#define	LINAC_H
+
+typedef enum
+{
+    nominal =  0,
+    open = 1,
+    closed = 2,
+    opening = 3,
+    closing = 4,
+    stopped_between_states =  5,
+    in_motion =  6,
+    stopped = 7
+} lin_actuator_states;
+
+
+lin_actuator_states moving = stopped;
```

Comment:
You want to put this in lin_actuator.c (and declare it static unless you have a good reason to do otherwise). The way it is now, this variable is public to all files that include lin_actuator.h which it shouldn't be.

Reply:
what does static actually mean?

Reply:
https://stackoverflow.com/questions/572547/what-does-static-mean-in-c

Reply:
in your situation, it declares the variable as local to that file

Reply:
it's a way to declare that that variable/function should not be modified by code that isn't in that file.